ctru-rs/examples/thread-locals.rs

```diff
@@ -1,10 +1,10 @@
 #![feature(horizon_thread_ext)]
 
-use ctru::prelude::*;
-
 use std::cell::RefCell;
 use std::os::horizon::thread::BuilderExt;
 
+use ctru::prelude::*;
+
```

Comment:
Oops, I didn't mean to commit all these changes but forgot I configured my rustfmt to `group_imports = "StdExternalCrate"` by default... hopefully it's not too distracting from the main contents of this PR

Reply:
The number files changed did seem a bit too large 😅

Reply:
I'll see if I can undo it to make it more clear what changed. I did end up touching a lot of files in order to add the test runner setup in doctests, which is how so many of them got reformatted (plus maybe I did a `cargo fmt` at one point).